src/main/java/org/sonar/plugins/scm/jazzrtc/JazzRtcBlameCommand.java

```diff
@@ -65,11 +69,18 @@ private void blame(FileSystem fs, InputFile inputFile, BlameOutput output) {
 
     int exitCode = execute(cl, consumer, stderr);
     if (exitCode != 0) {
-      throw new IllegalStateException("The jazz annotate command [" + cl.toString() + "] failed: " + stderr.getOutput());
+      if(ArrayUtils.contains(UNTRACKED_BLAME_RETURN_CODES, exitCode)) {
+        //skips file silently
+        return;
+      }
+      else {
```

Comment:
Another formatting issue that is not automatically fixed by Eclipse. "else" should be on the same line than closing "}".